src/openrct2/common.h

```diff
@@ -23,17 +23,16 @@
 #undef M_PI
 
 #ifdef _MSC_VER
-#include <time.h>
+#include <ctime>
 #endif
 
-#include <assert.h>
-#include <math.h>
-#include <stddef.h>
-#include <stdint.h>
-#include <stdio.h>
-#include <stdlib.h>
-#include <string.h>
-#include <stdbool.h>
```

Comment:
Note `cstdbool` include is gone, as it is a) not required b) deprecated in C++17